### PR TITLE
feat(proposal-cli): Support canisters outside the IC repository

### PR DESCRIPTION
**Pull request by gregorydemay**
Add support to the `proposal-cli` CLI tool for canisters outside the IC repository, such as the EVM RPC canister.

### CODE REVIEW

**Review by ninegua — APPROVED** [review]
LGTM! Thanks.

File: rs/cross-chain/proposal-cli/src/canister/mod.rs [hunk 1]
```diff
@@ -24,6 +25,7 @@ pub enum TargetCanister {
     IcpIndex,
     IcpLedger,
     LedgerSuiteOrchestrator,
+    EvmRpc,
```
Comment (ninegua) [inline rs/cross-chain/proposal-cli/src/canister/mod.rs:28]: In the future we can keep extending the enum, but at some point I wonder if generics (i.e. class and object implementing that class) start to make more sense?

Comment (gregorydemay) [reply rs/cross-chain/proposal-cli/src/canister/mod.rs:28]: >i.e. class and object implementing that class

@ninegua That's an interesting idea! Do you mean to be able to use it as a library and deal with canisters that this tool doesn't need to know about?

Comment (gregorydemay) [reply rs/cross-chain/proposal-cli/src/canister/mod.rs:28]: (I will merge it for now to be able to use it for the EVM RPC proposal, but happy to discuss this offline).

**Pull request merged**